components/FlexibleForms/StepForm.spec.tsx

```diff
@@ -86,23 +86,24 @@ describe('StepForm', () => {
     });
   });
 
-  it("also triggers the finish event if it's the only step", async () => {
-    render(
-      <StepForm
-        fields={mockFields}
-        onSubmit={() => true}
-        onFinish={mockFinish}
-        singleStep={true}
-      />
-    );
+  // TODO: bring this test back when we bring this functionality back
```

Comment:
next sprint we're gonna need this functionality, so it's commented out rather than deleted

Reply:
A thought – using `it.skip("...` might be better, as it gives us visibility on the fact it's skipped in our test output. A good reminder to go back and restore it later!

Reply:
sounds sensible to me!